stylesheets/print.styl

```diff
@@ -0,0 +1,139 @@
+body 
+	width 100% !important
+	margin 0 !important
+	padding 0 !important
+	line-height 1.45
+	font-size 14pt
+*:not(.news-item-default-image):not(.date-flag):not(.icon):not(.cd-ic),
+*:before,
+*:after,
+*:first-letter,
+p:first-line,
+div:first-line,
+blockquote:first-line,
+li:first-line 
+	background transparent !important
+    color #000 !important
+	box-shadow none !important
+	text-shadow none !important
+h1,h2,h3,h4,h5,h6 
+	page-break-after avoid
+h1 
+	font-size 19pt !important
+h2 
+	font-size 17pt !important
+h3 
+	font-size 15pt !important
+h4,h5,h6 
+	font-size 14pt !important
+p, h2, h3 
+	orphans 3
+	widows 3
+code 
+	font 12pt Courier, monospace
```

Comment:
Check w/ Sebastian to see if we have a preferred monospace font.

Reply:
he's ok with Courier